apps/datahub-e2e/cypress.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'cypress'
+import { nxE2EPreset } from '@nrwl/cypress/plugins/cypress-preset'
+
+export default defineConfig({
```

Comment:
I don't think that we the video.

Reply:
I think you're missing a word here!

Reply:
Yes we don't need the video :smile: 
We could disable it.

Reply:
added the param.